bindings/nodejs/lib/types/models/api/committee-response.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Bech32Address, EpochIndex } from '../../block';
+import { u64 } from '../../utils';
+
+/**
+ * Returns information of a validator (committee member).
+ */
+export interface CommitteeMember {
```

Comment:
Same here and next, for interfaces we have `I` prefix

Reply:
sure, but I think not consistently, which is why this might be a separate small PR, but can also do it here.

Reply:
Yeah that's then wrong in existing things, but they should be fixed separated

Reply:
Created an issue for the others #2044